pom.xml

```diff
@@ -81,17 +79,17 @@
 		<repository>
 			<id>spring-maven-release</id>
 			<name>Spring Maven Release Repository</name>
-			<url>https://maven.springframework.org/release</url>
+			<url>http://maven.springframework.org/release</url>
 		</repository>
 		<repository>
 			<id>jcenter-release</id>
 			<name>jcenter</name>
-			<url>https://oss.jfrog.org/artifactory/oss-release-local/</url>
+			<url>http://oss.jfrog.org/artifactory/oss-release-local/</url>
```

Comment:
Any reason these urls have been downgraded from the https protocol?

Reply:
I can't understand how this happened, i have upgraded.